### PR TITLE
Add responsive landing page, navbar

### PR DESCRIPTION
**Pull request by James-nyagatare**
#### What does this PR do?
-Creates the landing page and the navigation bar.

#### Description of Task to be completed?
- create navigation bar structure
- make navigation bar responsive
- create landing page structure
- make landing page responsive

#### Screenshots (if appropriate)
image

#### Questions:
N/A

### CODE REVIEW

**Review by harerakalex — CHANGES_REQUESTED** [review]
Well done @James-nyagatare. This PR looks nice to me, however, by considering your mockup you need to do minor improvement just to have the same picture as your mockup.  So pay attention to the design.

**Review by harerakalex — CHANGES_REQUESTED** [review]
One more quick feedback please try to put the content in the middle, I mean look at your mockup and compare the vertical alignment of it and the actual HTML page you have.

File: UI/css/styles.css [hunk 1]
```diff
@@ -159,12 +159,14 @@ nav {
   grid-template-columns: repeat(auto-fit, minmax(calc(500px / 2), 1fr));
 }
 .left {
+  margin-top: 6rem;
```
Comment (harerakalex) [inline UI/css/styles.css]: Still seems not to be in the middle of the page. Please make sure the margin-top is equal to the margin-bottom or use padding.
Screenshot from 2020-10-19 09-09-16

**Review by harerakalex — CHANGES_REQUESTED** [review]

File: UI/css/styles.css [hunk 2]
```diff
@@ -151,6 +151,7 @@ nav {
 }
 /* landing page  */
 #landing-page {
+  padding: 6.4rem 0;
```
Comment (harerakalex) [inline UI/css/styles.css]: Try to use percentage(hint 10%). padding: 10% 0;

**Review by harerakalex — CHANGES_REQUESTED** [review]

File: UI/css/styles.css (hunk 2 above)
Comment (James-nyagatare) [reply UI/css/styles.css]: image
I think it is working perfectly, Thanks for the feedback.

**Review by harerakalex — APPROVED** [review]

**Pull request merged**